### DISCUSSION TITLE
Can't install on latest version of NextJS (15.0.1)

### DISCUSSION THREAD

**Discussion by Jplus2** [chakra-ui/chakra-ui#9020] [General]
### Description

When running `npm i @chakra-ui/react @emotion/react` on freshly installed latest version of NextJS (15.0.1)
It will produce an error as shown by the screenshot below.

![image](https://github.com/user-attachments/assets/12eb5e87-0897-473c-bd89-46bbe4ba88c7)

### Link to Reproduction

https://codesandbox.io/p/devbox/chakra-ui-next-js-typescript-kxvyr

### Steps to reproduce

1. Run `npx create-next-app@latest`
2. Run `npm i @chakra-ui/react @emotion/react`

### Chakra UI Version

3.0.0

### Operating System

- [ ] macOS
- [X] Windows
- [ ] Linux

**Comment by zoolle** [comment]
you can try something like `npm install --legacy-peer-deps` until they will upgrade the react version in the package

**Comment by segunadebayo** [comment]
That peer-dep error was from 3.0.0. Current `@chakra-ui/react` installs on Next 15. Update the package.
